src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts

```diff
@@ -179,7 +180,6 @@ export class JJDisputeComponent implements OnInit {
         if (!this.lastUpdatedJJDispute.mostRecentCourtAppearance.jjSeized) this.lastUpdatedJJDispute.mostRecentCourtAppearance.jjSeized = 'N';
 
         // if (!this.isViewOnly) {
-        this.lastUpdatedJJDispute.mostRecentCourtAppearance.adjudicator = this.jjIDIR;
         this.lastUpdatedJJDispute.mostRecentCourtAppearance["adjudicatorName"] = this.jjName; // force to add property

```

Comment:
Is `adjudicatorName` still using ?

Reply:
Yes, I think that is still used for the logged in JJ.